CHANGELOG.md

```diff
@@ -6,10 +6,17 @@
 
 - improves default selection behavior for `LinkShapes.curvature` and
   `LinkShapes.line_dash`
+- adds `line_dash` to all 2D shapes
 - removes the significance of order in `ForceGraph.behaviors`
   - all node, link, and graph behaviors now have a (sensible default) `.rank` trait
     which determines the order in which they are applied.
   - lower `rank` are applied before higher `rank`
+- adds `DodoSource` for interpreting `doit` tasks graphs
+- adds `node_preserve_columns`, `link_id_column`, and `link_preserve_columns`
+  - these allow for keeping values when updating data, such as those created by the
```

Comment:
Just wanted to double check this was still going in this PR...

Reply:
yep, it's in here, and the demo shows _some_ improvement in the stability of the render, though the vanilla forces still encourage stuff to move